pkg/services/container.go

```diff
@@ -64,13 +64,24 @@ var Container = []dingo.Def{
 					cfg.RootCAs.AppendCertsFromPEM(ca)
 					klog.V(4).Infof("Added CA certificate %s", caPath)
 				}
+
+				tlsClientCert := os.Getenv("RABBITMQ_TLS_CLIENT_CERT")
+				tlsClientKey := os.Getenv("RABBITMQ_TLS_CLIENT_KEY")
+
+				if (tlsClientCert != "" && tlsClientKey != "") {
+					if cert, err := tls.LoadX509KeyPair(tlsClientCert, tlsClientKey); err == nil {
```

Comment:
This error is unhandled. I think we should at very least warn if this operation results in error, if not stop the program altogether as the user intention is to start the application with a valid cert, but an error occurred